whitelist.txt

```diff
@@ -6,3 +6,10 @@ pylint
 
 # Stands for 'source' package.
 src
+
+# Either a package's name or a part of it
+validators
+
+wtf
+
+wtforms
```

Comment:
Here and above we already can write that this is a package name, because I'm not sure that somebody will name a variable as `wtforms`. And actually I have no idea what `wtf` stands for in this package, so let's write something like this:
```
# part of the name of `flask-wtf` package
wtf

# `wtforms` python library
wtforms
```